src/core/constraint_template.jl

```diff
@@ -662,6 +662,17 @@ function constraint_storage_complementarity(pm::GenericPowerModel, i::Int; nw::I
     constraint_storage_complementarity(pm, nw, i)
 end
 
+""
+function constraint_storage_complementarity_mixed_int(pm::GenericPowerModel, i::Int; nw::Int=pm.cnw, cnd::Int=pm.ccnd)
+storage = ref(pm, nw, :storage, i)
+charge_lb = 0
+charge_ub = storage["charge_rating"]
+discharge_lb = 0
+discharge_ub = storage["discharge_rating"]
+constraint_storage_complementarity_mixed_int(pm, nw, i, charge_lb, charge_ub, discharge_lb, discharge_ub)
```

Comment:
Given that they are hard coded as zero, let's omit `charge_lb` and `discharge_lb`,  if we have a compelling use case later on the data model should be updated and then it will be easy to add this back in.

Reply:
style nit, add a space between data extraction and the function call.